Headers/DebugServer2/Target/POSIX/Thread.h

```diff
@@ -25,6 +25,12 @@ class Thread : public ds2::Target::ThreadBase {
   ErrorCode readCPUState(Architecture::CPUState &state) override;
   ErrorCode writeCPUState(Architecture::CPUState const &state) override;
 
+#if defined(OS_LINUX) && (defined(ARCH_X86) || defined(ARCH_X86_64))
```

Comment:
I think if this is going to be `#if defined(OS_LINUX)`, you should put it in `Linux/Thread.h` for now.